File: app/src/main/java/com/example/user/uberuber/Commons.java
package com.example.user.uberuber;

import com.example.user.uberuber.Remotes.IGoogleAPIService;
import com.example.user.uberuber.Remotes.RetrofitClient;

/**
 * Created by User on 7/4/2018.
 */

public class Commons {

    public static final String GOOGLE_API_URL = "https://maps.googleapis.com/";

    public static IGoogleAPIService getGoogleAPISerice()
    {
        return RetrofitClient.getClient(GOOGLE_API_URL).create(IGoogleAPIService.class);

    }


}

File: app/src/main/java/com/example/user/uberuber/Commons/Commons.kt
package com.example.user.uberuber.Commons

import com.example.user.uberuber.Remo.IGoogleAPIService
import com.example.user.uberuber.Remo.RetrofitClient

/**
 * Created by User on 17/4/2018.
 */
object Commons {
    private val GOOGLE_API_URL = "https://maps.googleapis.com/"

    val googleApiService:IGoogleAPIService
        get()= RetrofitClient.getClient(GOOGLE_API_URL).create(IGoogleAPIService::class.java)

}